pandas/core/groupby/groupby.py

```diff
@@ -377,14 +377,14 @@ def __unicode__(self):
 
     def _assure_grouper(self):
         """
-        we create the grouper on instantiation
-        sub-classes may have a different policy
+        We create the grouper on instantiation
+        sub-classes may have a different policy.
         """
         pass
 
     @property
     def groups(self):
-        """ dict {group name -> group labels} """
+        """ Dict {group name -> group labels} """
```

Comment:
Can you finish with period (besides having the text in a line between the quotes, as explained earlier).

Reply:
I actually had a question about this. My `awk` script did this one automatically, but this looks more like a description of what is being returned (I'm not familiar enough with the codebase to be sure of it). Is that the case or should I just go ahead and treat it as the docstring summary?

Reply:
this is the summary. In a different PR we can edit it and make the description look less cryptic, but for now jus add the period at the end, so the validation script does not complain

Reply:
:+1: